STAN.CLIENT/StanConnection.cs

```diff
@@ -469,61 +408,46 @@ public IStanSubscription Subscribe(string subject, EventHandler<StanMsgHandlerAr
         public IStanSubscription Subscribe(string subject, StanSubscriptionOptions options, EventHandler<StanMsgHandlerArgs> handler)
         {
             if (subject == null)
-                throw new ArgumentNullException("subject");
+                throw new ArgumentNullException(nameof(subject));
             if (options == null)
-                throw new ArgumentNullException("options");
+                throw new ArgumentNullException(nameof(options));
             if (handler == null)
-                throw new ArgumentNullException("handler");
+                throw new ArgumentNullException(nameof(handler));
 
             return subscribe(subject, null, handler, options);
         }
 
-        public IStanSubscription Subscribe(string subject, string qgroup,EventHandler<StanMsgHandlerArgs> handler)
+        public IStanSubscription Subscribe(string subject, string qgroup, EventHandler<StanMsgHandlerArgs> handler)
         {
             return Subscribe(subject, qgroup, AsyncSubscription.DefaultOptions, handler);
         }
 
         public IStanSubscription Subscribe(string subject, string qgroup, StanSubscriptionOptions options, EventHandler<StanMsgHandlerArgs> handler)
         {
             if (subject == null)
-                throw new ArgumentNullException("subject");
+                throw new ArgumentNullException(nameof(subject));
             if (qgroup == null)
-                throw new ArgumentNullException("qgroup");
+                throw new ArgumentNullException(nameof(qgroup));
             if (options == null)
-                throw new ArgumentNullException("options");
+                throw new ArgumentNullException(nameof(options));
             if (handler == null)
-                throw new ArgumentNullException("handler");
+                throw new ArgumentNullException(nameof(handler));
 
             return subscribe(subject, qgroup, handler, options);
         }
 
         public void Close()
         {
-            Msg reply = null;
+            if (IsClosed)

```

Comment:
@harold-toledo the definition of IsClosed would allow this to be called multiple times if LeaveOpen is true.

Reply:
I do not see how LeaveOpen could potentially affect this call, and the fact that we could potentially call it multiple times should not cause any issues.